package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "eslint-plugin-enterprise-extras",
+  "name": "@buildertrend/eslint-plugin-enterprise-extras",
   "description": "Extra eslint rules for enterprise environments focusing on React and Typescript",
   "version": "1.1.2",
```

Comment:
Do we need a version bump? Or what will the process be on that?

Reply:
Because this gets published under a different package, it doesn't require a version bump. Otherwise, version bumps are needed whenever files from the `dist/` folder ar changed